locales/fr.json

```diff
@@ -3,13 +3,40 @@
         "commands" : "Dis: \"Que puis-je dire?\" pour voir une liste de commandes..."
     },
     "traffic" : {
-        "time_to" : "Temps de {{name}}: "
+        "time_to" : "Temps de trajet : {{name}}: "
```

Comment:
I don't speak French, but is this grammatically correct? This will result in some strange looking double ':', for example

```
Temps de trajet : travail: 20min
```

This is inconsistent with the rest of the lenguages